tools/utils.py

```diff
@@ -329,3 +336,22 @@ def get_cluster_server(cluster_name='default'):
         )
 
     return out
+
+
+def override_service_type_definition_and_node_port(
+    internal_definitions_path: str,
```

Comment:
is this source manifest filaname and output filename?

Reply:
Yes

Reply:
Can we give them a better name? :sweat_smile: It's not really clear to me, I had to follow the code to guess it